code/pizzabot.js

```diff
@@ -5,3 +5,38 @@ const pepperoni = "Pepperoni Pizza"
 const pizzaPrice = 80
 
 //Put your Javscript code here:
+
+let checkOrderName = (orderName) => {
+  if (
+    orderName === vegetarian || orderName === hawaiian || orderName === pepperoni
+  ) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+let orderTotal = (orderQuantity) => {
+  return +orderQuantity * pizzaPrice
+}
+
+let pizzaTime = (orderQuantity) => {
+  if (+orderQuantity <= 2) {
+    return 10
+  } else if (+orderQuantity <= 5) {
+    return 15
+  } else {
+    return 20
+  }
+}
+
+alert(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}`)
+
+let orderName = prompt(`Enter the name of the pizza you want to order today.`)
+
+if (checkOrderName(orderName) === true) {
```

Comment:
This means `true` as well! If you want to check `false` you can type `!checkOrderName(orderName)`!
```suggestion
if (checkOrderName(orderName)) {
```

Reply:
Thanks! That's much more clear, I think I used something similar in the cluedo projects without the === true.